man/systemd.link.xml

```diff
@@ -548,6 +548,15 @@
           cleared.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>WakeOnLanSecureOnPassword=</varname></term>
```

Comment:
i'd probably just name this "WakeOnLanPassword=". i.e. no need to repeat the trademark here, the generic naming as just "password" is clear enough.